core/src/main/java/com/novoda/noplayer/model/AudioTracks.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.model;
+
+import com.novoda.noplayer.internal.exoplayer.mediasource.AudioTrackType;
+import com.novoda.utils.Optional;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class AudioTracks implements Iterable<PlayerAudioTrack> {
+
+    private final List<PlayerAudioTrack> audioTracks;
+
+    public static AudioTracks from(List<PlayerAudioTrack> audioTracks) {
+        return new AudioTracks(Collections.unmodifiableList(audioTracks));
+    }
+
+    private AudioTracks(List<PlayerAudioTrack> audioTracks) {
+        this.audioTracks = audioTracks;
+    }
+
+    public boolean containsPlayerAudioTrackWith(AudioTrackType trackType) {
```

Comment:
not `contains(Type)` because this is a collection of `AudioTrack` not `AudioType`.

we discussed also offline that if a manifest contains multiple languages, that there will be multiple tracks with the same type - e.g. "english - main" and "french - main" at which point `containsTrackWith(AudioTrackType)` is not enough information?